src/intersection-observer.js

```diff
@@ -208,6 +209,10 @@ export class IntersectionObserver extends Observable {
    * @private
    */
   flush_() {
+    if (!this.isElementInDoc_()) {
```

Comment:
Well missed @dvoytenko comment here. checking `contains` is cheap, but I assume getting `this.baseElement_.win.document` is expensive?